src/components/Icon/Icon.tsx

```diff
@@ -12,7 +12,35 @@ const isSquare = new Set([
   'youtube'
 ]);
 
+const isNumber = new Set([
```

Comment:
Can we reuse [numberIcons](https://github.com/cfpb/design-stories/pull/107/files#diff-66bc180de4d05036897534cd2d978a654b44b2eb27dd5fce306503b25c69e502R53) to avoid duplicating this list?